### PR TITLE
 fa: quick fix for "missed space and comma in the main page headline" and HTML in releases page

### PR DESCRIPTION
**Pull request by m-esm**

### CODE REVIEW

File: locale/fa/download/releases.md [hunk 1]
```diff
@@ -9,16 +9,12 @@ modules: "<code>NODE_MODULE_VERSION</code> refers to the ABI (application binary
...
-        <li><a href="https://nodejs.org/dist/latest-v10.x/">Node.js 10.x</a></li>
-        <li><a href="https://nodejs.org/dist/latest-v8.x/">Node.js 8.x</a></li>
-        <li><a href="https://nodejs.org/dist/latest-v6.x/">Node.js 6.x</a></li>
-        <li><a href="https://nodejs.org/dist/latest-v4.x/">Node.js 4.x</a></li>
-        <li><a href="https://nodejs.org/dist/latest-v0.12.x/">Node.js 0.12.x</a></li>
-        <li><a href="https://nodejs.org/dist/latest-v0.10.x/">Node.js 0.10.x</a></li>
-        <li><a href="https://nodejs.org/dist/">تمامی نسخه‌ها</a></li>
-    </ul>
-</div>
+###  به دنبال آخرین انتشار از شاخه ورژن می‌گردید؟
```
Comment (fhemberger) [inline locale/fa/download/releases.md]: Please use the HTML provided on `master`.

**Review by fhemberger — CHANGES_REQUESTED** [review]

File: locale/fa/download/releases.md (hunk 1 above)
Comment (m-esm) [reply locale/fa/download/releases.md]: replaced.

**Review by ebraminio — APPROVED** [review]

File: locale/fa/about/index.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ Node مدل رویداد را کمی به جلوتر می‌برد و  [event lo
  
 HTTP در Node شهروند درجه یک محسوب می‌شود، طراحی شده با درنظر گیری streaming و زمان تأخیر کم. این امر Node را به یک گزینه مناسب برای بنیاد کتابخانه وب و یا یک چارچوب تبدیل می‌کند.
 
-فقط به خاطر اینکه Node بدون threads طراحی شده، به این معنی نیست که نتوانید از چند هسته در مخیط خود بهره ببرید. فرایند‌های زیرن می‌توانند توسط ‌[`child_process.fork()`][] API  اجرا شوند. فرایند های زیرین به شکلی طراحی شده‌اند که تعامل کردن با آن‌ها به سادگی انجام شود. 
+فقط به خاطر اینکه Node بدون Threadها طراحی شده، به این معنی نیست که نتوانید از چند هسته در محیط خود بهره ببرید. فرایند‌های زیرن می‌توانند توسط ‌[`child_process.fork()`][] API  اجرا شوند. فرایند های زیرین به شکلی طراحی شده‌اند که تعامل کردن با آن‌ها به سادگی انجام شود.
```
Comment (AliSawari) [inline locale/fa/about/index.md]: I think you meant to say زیرین Here

**Review by AliSawari — CHANGES_REQUESTED** [review]

File: locale/fa/about/index.md (hunk 2 above)
Comment (AliSawari) [reply locale/fa/about/index.md]: exactly here 
فرایند‌های زیرن می‌توانند توسط

**Comment by m-esm** [comment]
@fhemberger hi again, edits committed :open_hands:

**Review by AliSawari — APPROVED** [review]
LGTM :+1:

**Pull request merged**